### DISCUSSION TITLE
Read hooks

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#5215] [Feature Requests]
Hey everyone,

Maybe its nice to add read hooks, basicly the reason why should be so you can log when someone has viewed a record, for example in finacial cases, this could be usefull.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/4847, https://github.com/directus/directus/discussions/3736, https://github.com/directus/directus/discussions/4812#discussioncomment-560575

**Comment by benhaynes** [comment]
Closing this as a duplicate of https://github.com/directus/directus/discussions/3736
